src/auth/crypto.cpp

```diff
@@ -271,6 +297,16 @@ auto AsString(PasswordHashAlgorithm hash_algo) -> std::string_view {
   return *utils::EnumToString<PasswordHashAlgorithm>(hash_algo, password_hash_mappings);
 }
 
+auto HashSize(PasswordHashAlgorithm hash_algo) -> size_t {
+  switch (hash_algo) {
+    case PasswordHashAlgorithm::BCRYPT:
+      return 60;  // NOTE: BCRYPT_HASHSIZE is 64, but the result is actually 60B
+    case PasswordHashAlgorithm::SHA256:
+    case PasswordHashAlgorithm::SHA256_MULTIPLE:
+      return SHA::SHA_LENGTH;
+  }
```

Comment:
Return a struct for unsalted vs salted sizes. Or make a something obvious these are unsalted lengths.